From 94d4af20292e41798a120911690766a760eb901a Mon Sep 17 00:00:00 2001
From: Chetan Surpur <chetan51@gmail.com>
Date: Thu, 30 Dec 2010 15:16:17 -0800
Subject: [PATCH] Switched to offline mode

---
 src/views/base.jade |    8 +++++---
 1 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/views/base.jade b/src/views/base.jade
index 5bb12c1..aac5654 100644
--- a/src/views/base.jade
+++ b/src/views/base.jade
@@ -2,7 +2,7 @@
 html
   head
     title= title
-    link(href="http://fonts.googleapis.com/css?family=UnifrakturCook:bold", rel="stylesheet", type="text/css")
+    //link(href="http://fonts.googleapis.com/css?family=UnifrakturCook:bold", rel="stylesheet", type="text/css")
     link(href=base_url+"includes/css/reset.css", rel="stylesheet", type="text/css")
     link(href=base_url+"includes/css/typography.css", rel="stylesheet", type="text/css")
     link(href=base_url+"includes/css/colors.css", rel="stylesheet", type="text/css")
@@ -12,10 +12,12 @@ html
     // LESS
     script(src=base_url+"includes/js/less-1.0.35.min.js", type="text/javascript")
     // jQuery Library + UI Tools
-    script(src="http://cdn.jquerytools.org/1.2.5/full/jquery.tools.min.js", type="text/javascript")
+    //script(src="http://cdn.jquerytools.org/1.2.5/full/jquery.tools.min.js", type="text/javascript")
+    script(src=base_url+"includes/js/jquery.tools.min.js", type="text/javascript")
     // jQuery UI
     link(href=base_url+"includes/css/ui-darkness/jquery-ui-1.8.7.custom.css", rel="stylesheet", type="text/css")
-    script(src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.7/jquery-ui.min.js", type="text/javascript")
+    //script(src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.8.7/jquery-ui.min.js", type="text/javascript")
+    script(src=base_url+"includes/js/jquery-ui-1.8.7.custom.min.js", type="text/javascript")
     script(src=base_url+"includes/js/client.js", type="text/javascript")
   body
     #header
-- 
1.7.2.3

